test-utils/runtime/src/lib.rs

```diff
@@ -388,6 +389,12 @@ impl From<frame_system::Event<Runtime>> for Event {
 	}
 }
 
+impl From<pallet_balances::Event<Runtime>> for Event {
```

Comment:
You can probably use `decl_event` macro instead.

Reply:
actually this code is no longer needed